invenio/modules/access/local_config.py

```diff
@@ -24,12 +24,14 @@
 
 # pylint: disable=C0301
 
-from invenio.config import CFG_SITE_NAME, CFG_SITE_URL, CFG_SITE_SECURE_URL, CFG_SITE_SUPPORT_EMAIL, CFG_SITE_RECORD, CFG_SITE_ADMIN_EMAIL
 from invenio.base.i18n import _
-from invenio.base.globals import cfg as config
+from invenio.base.config import CFG_SITE_URL, CFG_SITE_RECORD, \
```

Comment:
They are used inside this file to create message for old admin interface:

e.g. https://github.com/hachreak/invenio/blob/3106_deprecate_invenio_config/invenio/modules/access/local_config.py#L228

I'll delete all no necessary import in my branch that make a refactoring of [access module](https://github.com/hachreak/invenio/tree/3299_access_module_refactoring).